src/otx/core/metrics/kitti_3d_eval/eval.py

```diff
@@ -0,0 +1,811 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""KITTI 3D eval for OTX."""
+
+# flake8: noqa
+# mypy: ignore-errors
+
+from __future__ import annotations
+
+import io as sysio
+from typing import Any
+
+import numba
+import numpy as np
+import torch
+
+if torch.cuda.is_available():
+    from .rotate_gpu_iou import rotate_iou_eval_gpu as rotate_iou_eval
+else:
+    from .rotate_iou import rotate_iou_eval_cpu as rotate_iou_eval
+
+
+@numba.jit(nopython=True)
+def get_thresholds(
+    scores: np.ndarray,  # 1D array of confidence scores
```

Comment:
```suggestion
    scores: np.ndarray,
```
Could you remove comment next to argument? It's duplication of docstring beolow

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4031/commits/00200f4d4811d82e86e371cb5dabd2541098e83c